src/api/Web.Bff.Shopping/appsettings.json

```diff
@@ -1,6 +1,6 @@
 {
   "urls": {
-    "catalog": "http://localhost:1"
+    "catalog": "https://localhost:44391"
```

Comment:
This should be definitely undone. The reason for it is that we are having our appsettings.json just as an example, so adding a proper URL even for local testing is a bad habit. The only changes that should be made in appsettings.json - example update when config was restructured.
- [ ] remove changes made in appsettings.json

Also, for the sake of consistency, I would like to ask you to add ending slash in your appsettings.development.json and also remove starting slash on CommentsApiOperation strings.